media-tv/plex-media-server/files/plexmediaserver.desktop.patch

```diff
@@ -0,0 +1,8 @@
+--- /usr/share/applications/plexmediaserver.desktop.org	2020-05-23 01:39:34.105959787 -0500
```

Comment:
Let's trim patches where we can - drop the date [parts](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto). We also usually use patch level 1, so we should prefix the paths with a/ and b/ respectively.

Reply:
Fixed